MIP/mip-34/README.md

```diff
@@ -0,0 +1,270 @@
+- **MCR** - Multi-commit Rollup : an implementation of the postconfirmation in FFS. See [Overview](#overview)  
+- **L2-finality certificate** - See [Overview](#overview)
+- **L2-confirmation** - a finality guarantee related to L2. See [Overview](#overview)
+- **postconfirmation** - a finality guarantee related to L1. See [Overview](#overview)  
+- **L1-finality** - finality mechanism for layer 1. 
+- **L2-finality** -  finality mechanism (confirmation) for layer 2.
+- **PoS** - Proof of Stake
+
+In addition we make the note for the following terms:
+
+- **batch** (not recommended)
+Less clean, but more common term for sequencer-batch. May be mixed up with the batch of transactions sent to the sequencer, or with the batch of blocks that should be processed by the L1-contract. 
+- **block** 
+More common term for L2-block. May be mixed up with the batch of transactions sent to the sequencer, the L1-block or with the batch of L2-blocks that should be processed by the L1-contract. Here we mean L2-block when we say block.
+- **attester**  (not recommended)
+The term attester has been deprecated in favor of validator.
+- **quorum certificate** (not recommended)
+The term quorum certificate has been deprecated in favor of L2-finality certificate.
+
+
+
+## Motivation
+
+Layer 2s (L2), including rollups, publish or secure transaction data in a data availability (DA) layer or at Ethereum mainnet (Layer 1, L1). Validity and optimistic rollups can finalize (confirm) transactions within approximately 30 minutes, resp. ~1 week. Until a transaction is finalized, there is no assurance about its validity and result (success or failure). This can be a limiting factor for certain types of DeFi applications.
+
+Our objective is to enable transaction issuers to quickly get some guarantees that their transactions are correctly included in a block. The crypto-economic security is provided by a PoS protocol.
+
+The mechanism can be deployed independently for a chain, or used in combination with existing settlement mechanisms, such as ZK and optimistic settlements.
+
+As a result, users can rely and trust the **L2-finality**  to use as confirmation, or if the chain is configured to do so, wait for **L1-finality**, such as end of challenge window for fraud proofs (optimistic L2) or verification of a ZK-proof (validity L2).
+
+A introduction to FFS can be found in [this blog post on Fast-Finality Settlement](https://blog.movementlabs.xyz/article/security-and-fast-finality-settlement). A more detailed description of a (partial) implementation of the mechanism is available at [this blog post on postconfirmations](https://blog.movementlabs.xyz/article/postconfirmations-L2s-rollups-blockchain-movement).
+
+This MIP provides an overview of an architecture of FFS, and its main components.
+
+## Specification
+
+### Overview
+
+The objective of FFS is to confirm that transactions are processed correctly. It does not relate to the ordering of transactions.
```

Comment:
Naturally, however, it is also confirming that participants have read the correct order of transactions.

Reply:
yes. i have added this